scripts-available/CDB_Overviews.sql

```diff
@@ -904,14 +808,20 @@ AS $$
       columns := columns || ', n AS _feature_count';
     END IF;
 
-    EXECUTE Format('DROP TABLE IF EXISTS %I.%I CASCADE;', schema_name, overview_rel);
+    overview_table_name := Format('%I.%I', schema_name, overview_rel);
```

Comment:
We should extract this somehow and make the *_Reduce_Strategy functions return the data not make the tables, populate them and so on. I didn't want to change this becuase is a huge change...